public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -121,6 +121,15 @@ export class DashboardModel implements TimeModel {
     originalDashboard: true,
   };
 
+  static autoMigratePluginIDs = [
+    'graph',
+    'table-old',
+    'grafana-piechart-panel',
+    'grafana-worldmap-panel',
+    'singlestat',
+    'grafana-singlestat-panel',
+  ];
```

Comment:
This list already exists in the code here :) https://github.com/grafana/grafana/blob/01fb2cff624e402b927e16f4e04e9fc35c7ab08c/public/app/features/dashboard/state/PanelModel.ts#L133

Reply:
Spot on :+1: Fixed in 3ce974089bffbed30c9f4c65fece85547152b2be